CODEFLARE.md

```diff
@@ -330,3 +330,36 @@ is based solely on considering the currently admitted workloads of
 just those cluster queues that are exceeding their nominal
 quota. Workloads admitted by cluster queues that are currently at or
 below their nominal quota will not be preempted.
+
+## Allowed Kinds
+
+MLBatch allows users to directly create the following Kinds of compute
+resources:
+   + AppWrapper
+   + PyTorchJob (allowed, but recommend to put inside an AppWrapper)
+   + RayJob (allowed, but recommend to put inside an AppWrapper)
+   + RayCluster (allowed, but recommend to put inside an AppWrapper)
+
+MLBatch also allows users to directly create the following Kinds of
+non-compute resources:
+   + Service
+   + Secret
+   + ConfigMap
+   + PersistentVolumeClaim
+   + PodGroup (allowed, but recommend to put inside an AppWrapper)
+
+MLBatch allows users to wrap an arbitrary number of one or more of the
+following Kinds inside of an AppWrapper:
+   + PyTorchJob
+   + RayJob
+   + RayCluster
+   + Deployment
+   + StatefulSet
+   + Pod
+   + Job
+   + ServiceAccount
+   + Service
+   + Secret
+   + ConfigMap
+   + PersistentVolumeClaim
+   + PodGroup
```

Comment:
The codeflare operator's RBACs actually allow a few more non-compute Kinds (Ingress, Route), but I elided them from the user docs as I don't think we want them.

Reply:
I agree. These is a permission "leak". The Kubernetes flavor of MLBatch does not support these unwrapped kinds.